modules/account-lib/src/coin/xrp/utils.ts

```diff
@@ -1,10 +1,16 @@
+import { RippleAPI } from 'ripple-lib';
+import * as RippleAddressCodec from 'ripple-address-codec';
 import { BaseUtils } from '../baseCoin';
 import { NotImplementedError } from '../baseCoin/errors';
 
+export const initApi = (): RippleAPI => new RippleAPI({});
```

Comment:
Is this offline? We should never use an online service since this library MUST run offline.

Reply:
This is offline